sources/packages/backend/libs/sims-db/src/pagination-utils.ts

```diff
@@ -18,6 +18,19 @@ export function transformToApplicationEntitySortField(
     return orderByCondition;
   }
 
+  if (sortField === "dateSubmitted") {
+    orderByCondition["applicationOfferingChangeRequest.createdAt"] = sortOrder;
+    return orderByCondition;
+  }
+
+  if (sortField === "status") {
```

Comment:
createdAt must be the second level of sort even while sorting by application number and full name right ?

Reply:
We don't have sorting by application number and full name in this ticket.